Sources/ScipioKit/Producer/PIF/PIFGenerator.swift

```diff
@@ -1,308 +1,208 @@
-            fatalError("Resolved Target named \(c99Name) is not found.")
-        }
-
-        guard let resolvedPackage = descriptionPackage.graph.package(for: resolvedTarget) else {
-            fatalError("Could not find a package")
+        for index in 0..<target.buildConfigurations.count {
+            updateBuildConfiguration(&target.buildConfigurations[index], target: target, sdk: sdk)
         }
-
-        self.resolvedTarget = resolvedTarget
-        self.resolvedPackage = resolvedPackage
-    }
-
-    private var c99Name: String {
-        resolvedTarget.c99name
-    }
-
-    func modify() -> PIF.Target {
-        updateLibraryTargetSettings()
-
-        return pifTarget
     }
 
-    private func updateLibraryTargetSettings() {
-        pifTarget.productType = .framework
-
-        let newConfigurations = pifTarget.buildConfigurations.map(updateBuildConfiguration)
-
-        pifTarget.buildConfigurations = newConfigurations
-    }
-
-    private func updateBuildConfiguration(_ original: PIF.BuildConfiguration) -> PIF.BuildConfiguration {
-        var configuration = original
-        var settings = configuration.buildSettings
-        let name = pifTarget.name
-        let c99Name = name.spm_mangledToC99ExtendedIdentifier()
-
+    private func updateBuildConfiguration(_ configuration: inout PIFKit.BuildConfiguration, target: PIFKit.Target, sdk: SDK) {
+        let name = target.name
         let toolchainLibDir = (try? buildParameters.toolchain.toolchainLibDir) ?? .root
 
-        settings[.PRODUCT_NAME] = "$(EXECUTABLE_NAME:c99extidentifier)"
-        settings[.PRODUCT_MODULE_NAME] = "$(EXECUTABLE_NAME:c99extidentifier)"
-        settings[.EXECUTABLE_NAME] = c99Name
-        settings[.TARGET_NAME] = name
-        settings[.PRODUCT_BUNDLE_IDENTIFIER] = name.spm_mangledToBundleIdentifier()
-        settings[.CLANG_ENABLE_MODULES] = "YES"
-        settings[.DEFINES_MODULE] = "YES"
-        settings[.SKIP_INSTALL] = "NO"
-        settings[.INSTALL_PATH] = "/usr/local/lib"
-        settings[.ONLY_ACTIVE_ARCH] = "NO"
-
-        settings[.GENERATE_INFOPLIST_FILE] = "YES"
+        configuration.buildSettings["PRODUCT_NAME"] = "$(EXECUTABLE_NAME:c99extidentifier)"
+        configuration.buildSettings["PRODUCT_MODULE_NAME"] = "$(EXECUTABLE_NAME:c99extidentifier)"
+        configuration.buildSettings["EXECUTABLE_NAME"] = .string(target.c99Name)
+        configuration.buildSettings["TARGET_NAME"] = .string(name)
+        configuration.buildSettings["PRODUCT_BUNDLE_IDENTIFIER"] = .string(name.spm_mangledToBundleIdentifier())
+        configuration.buildSettings["CLANG_ENABLE_MODULES"] = true
+        configuration.buildSettings["DEFINES_MODULE"] = true
+        configuration.buildSettings["SKIP_INSTALL"] = false
+        configuration.buildSettings["INSTALL_PATH"] = "/usr/local/lib"
+        configuration.buildSettings["ONLY_ACTIVE_ARCH"] = false
+        configuration.buildSettings["SDKROOT"] = .string(sdk.settingValue)
```

Comment:
`SDKROOT` is added, for what reason is that required?

Reply:
dumppif for `.xcodeproj` generates this value, so I added, but we needn't set the value. I'll revert this.